component/main.jsonnet

```
@@ -1,26 +1,39 @@
 local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
+local rl = import 'lib/resource-locker.libjsonnet';
+
 local inv = kap.inventory();
 local params = inv.parameters.openshift4_monitoring;
 
 local rules = import 'rules.jsonnet';
 
+local ns =
+  if params.namespace != 'openshift-monitoring' then
+    error 'Currently, this component does not support values for parameter\nnamespace other than "openshift-monitoring".'
```

Question, how is this printed in the CLI? Is the intention to wrap the line like
```
Currently, this component does not support values for parameter
namespace other than "openshift-monitoring".
```
or did you intend to give reference to a parameter in the hiarchy?

```suggestion
    error 'Currently, this component does not support values for parameter.namespace other than "openshift-monitoring".'
```

---

The message is printed with a line break on the CLI:

```
Jsonnet error: failed to compile /home/simon/work/syn/commodore/vendor/openshift4-monitoring/component/main.jsonnet:
 RUNTIME ERROR: Currently, this component does not support values for parameter
namespace other than "openshift-monitoring".
```
However, I did not intend to have the line break in the message, that must have slipped in due to my Jsonnet editor plugin.